css-layout-classes/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>css layout classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-third">
+        <img class="characters" src="./images/link.png" alt="link" />
+      </div>
+      <div class="column-third">
+        <h2 class="column-third">Super Smash Bros. Ultimate</h2>
+        <p class="column-third">
+          Super Smash Bros. Ultimate received universal acclian from both
+          critics and players, with some critics calling it the best installment
+          in the series. It recevied praise for its large amount of content and
+          fine-tuning of exsiting Samsh gameplay elements, although its online
+          mode was widely criticized
+        </p>
+      </div>
+      <div class="column-third">
+        <img class="characters" src="./images/yoshi.png" alt="yoshi" />
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-fourth">
+        <img class="characters" src="./images/kirby.png" alt="kirby" />
+      </div>
+      <div class="column-fourth">
+        <img class="characters" src="./images/fox.png" alt="fox" />
+      </div>
+      <div class="column-fourth">
+        <img class="characters" src="./images/pikachu.png" alt="pikachu" />
+      </div>
+      <div class="column-fourth">
+        <img class="characters" src="./images/ness.png" alt="ness" />
+      </div>
+    </div>
+    <div class="row">
+      <div class="two-thirds">
+        <h1 class="two-thirds">Donkey Kong</h1>
+        <div class="par1-2">
+          <p class="two-thirds">
+            Donkey Knong is a series of video games featuring the adventures of
+            a gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981.
+            The franchise consists mainly of two game genres but also includes
+            additional spin-off titles of various genres.<br /><br />The games
+            of the first genre are mostly single-screen platform/action puzzle
+            types, featuring Donkey Kong as the opponent in an industrial
+            construction setting.
+          </p>
+          <div class="two-thirds">
+            <img class="donkey" src="./images/dk.png" alt="donkey kong" />
+          </div>
```

Comment:
Everything should not be inside of your `two-thirds` class. You should have 2 sibling elements, one with a `column-third` & another one with a `column-two-thirds`

`.row` > `.column-third` & `.column-two-thirds`. And then your content for the columns would go inside their respective column